xml/schema/Campaign/Survey.xml

```diff
@@ -214,6 +214,13 @@
     <default>NULL</default>
     <comment>Used to store option group id.</comment>
```

Comment:
@eileenmcnaughton I'm pretty sure the comment here is incorrect. This field appears to be used to store the option **value** of the option group configured for this survey.
![Screenshot from 2021-09-02 21-21-58](https://user-images.githubusercontent.com/2874912/131936342-ce681f33-3512-4a08-b1a1-514ef59779e7.png)

Reply:
@colemanw I created a survey & got a value of 106 in that field - it correlated to the option group id - then when you phone people up you create activities & they have a field (result?) that maps to the option value

Reply:
Oh, no, wait, I get it. This stores the option_group_id of the option group being displayed on this form.